market/orderTypes/limit.ts

```
@@ -0,0 +1,54 @@
+function findMatchingBuyOrder(sellOrder: Order): Order | null {
+    const buyOrdersMarket = getBuyActiveOrdersByCollectibleAndType(sellOrder.collectible_id, 'MARKET');
+    const buyOrdersLimit = getBuyActiveOrdersByCollectibleAndType(sellOrder.collectible_id, 'LIMIT');
+    
+    if (buyOrdersMarket.length > 0) {
+        const buyOrderMarket = buyOrdersMarket[0];
+        return buyOrderMarket;
+    }
+
+    buyOrdersLimit.sort((x, y) => y.price - x.price);
+
+    for (let buyOrder of buyOrdersLimit) {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'buyOrder' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)